go/oprf/oprf_test.go

```diff
@@ -52,6 +54,11 @@ func TestFullVOPRFP521Multiple(t *testing.T) {
 	checkFull(t, validOPRFP521Ciphersuite, 5)
 }
 
+func TestFullCurve448(t *testing.T) {
+	t.Skip("Skipping test.")
```

Comment:
I guess skipping this test is related to the problems with `Blind`?

Reply:
Yes, precisely ;).